kalite/facility/templates/facility/facility_user.html

```diff
@@ -74,7 +27,12 @@ <h3 class="no-margin-top">{{ title }}</h3>
                     {{ field.label_tag }}
                     {{ field }}
                     {{ field.errors }}
-                    <span class="helptext">{{ field.help_text }}</span>
+                    {% if field.id_for_label == "id_group" and request.is_admin %}
+                        {# Allow admins to add a new group #}
+                        <span class="helptext">(<a href="{% url 'add_group' %}?facility={{facility.id}}&next={{ request.get_full_path|urlencode }}">{% trans "Add a new group" %}</a>)</span>
```

Comment:
Searching the page for '&' highlights all instances of that character on the page, so if it is a weird unicode character, it always was.